src/TheGuardianIn2020Banner/index.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { ReactElement } from 'react';
+import { withKnobs, text } from '@storybook/addon-knobs';
+import { BrazeMessage } from '../BrazeMessage';
+import { StorybookWrapper } from '../utils/StorybookWrapper';
+import { knobsData } from '../utils/knobsData';
+
+export default {
+    component: 'TheGuardianIn2020Banner',
+    title: 'Components/TheGuardianIn2020Banner',
+    decorators: [withKnobs],
+    parameters: {
+        knobs: {
+            escapeHTML: false, // Block HTML escaping, preventing double-escaping of imgUrl special characters in Storybook
+        },
+    },
+};
+
+export const defaultStory = (): ReactElement => {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const slotName = text('slotName', 'Banner');
+    const header = text('header', 'The Guardian’s impact in 2020');
+    const body = text(
+        'body',
+        'Thanks to your generous support in this extraordinary year, our open, independent journalism was read by millions. From the pandemic to our urgent coverage of the climate crisis, Black Lives Matter protests and the US election, our reporting had a powerful impact.',
+    );
+    const componentName = text('componentName', 'TheGuardianIn2020Banner');
+
+    // This is to make the data available to the guPreview add-on:
+    knobsData.set({ header, body, componentName });
+
+    return (
+        <StorybookWrapper>
+            <BrazeMessage
+                componentName={componentName}
+                logButtonClickWithBraze={(internalButtonId) => {
+                    console.log(`Button with internal ID ${internalButtonId} clicked`);
```

Comment:
Are we ok to leave these `console.log` lines?

Reply:
Yeah I think this is fine. In the context of Storybook there's not much else we can do in these handlers, so we just log something so it's possibility to get visibility of them.